backend/tests_configs.ini

```diff
@@ -202,6 +202,7 @@ email.notification.enabled_on_invitation = False
 webdav.ui.enabled = False
 webdav.base_url = https://localhost:3030
 webdav.root_path = /webdav
+debug = True
```

Comment:
Should be removed here no ?

Reply:
Again I forgot it… Is there a way to set this config parameter through an environment variable ? I find it quite useful to get the backtrace when an http api responds a 500 error.

Reply:
Yes, like most parameters of Tracim.
Set TRACIM_DEBUG.

Reply:
OK, thank your for the tip.